### PR TITLE
Update extension for manifest v3

### PR DESCRIPTION
**Pull request by EthanLuisMcDonough**
Chrome's deprecation of manifest v2 extensions is coming up.  We'll need to update the extension to manifest v3 by January [if we want to keep our featured badge](https://developer.chrome.com/docs/extensions/mv3/mv2-sunset/).

**Manifest v3 updates**
* Update manifest version
* Remove deprecated chrome functions
* Change setTimeout in background.ts to chrome alert API to abide by new manifest v3 spec
* Update chrome types

**Bugfixes**
* Fix username bug on profile page
* Allow graphql helper function to be called in background.ts

**Refactoring**
* Lint
* Change location.href to location.path in split to avoid query strings ruining our profile page logic

**Meta**
* Update license year
* Add new contributors to readme

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 KA Extension Developers
+Copyright (c) 2023 KA Extension Developers
```
Comment (MatthiasPortzel) [inline LICENSE:3]: :thinking:

**Review by MatthiasPortzel — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (EthanLuisMcDonough) [reply LICENSE:3]: I figure its better to be ahead of the game given I don't want to go back and update it later.

Comment (MatthiasPortzel) [reply LICENSE:3]: But it's wrong...

File: manifest.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-	"manifest_version": 2,
+	"manifest_version": 3,
 	"name": "The Khan Academy Extension",
 	"short_name": "KA Extension",
 	"author": "Khan Academy Extension Developers",
 	"homepage_url": "https://github.com/ka-extension",
 	"version": "4.7.14",
 	"description": "A Browser Extension for Khan Academy that adds more features to the site",
-	"minimum_chrome_version": "51.0.2704",
+	"minimum_chrome_version": "65.0.3299",
```
Comment (lukekrikorian) [inline manifest.json:9]: It might be helpful to know for posterity why this version was chosen

**Review by lukekrikorian — APPROVED** [review]

File: manifest.json (hunk 2 above)
Comment (EthanLuisMcDonough) [reply manifest.json:9]: Minimum chrome version that supports async generators.  Esbuild isn't able to rewrite those yet.

**Pull request merged**